app/src/main/res/values/strings.xml

```diff
@@ -78,4 +78,9 @@
 
     <!-- Content description (not visible, for screen readers etc.): Reload current website -->
     <string name="content_description_reload">Reload website</string>
+
+    <string name="firstrun_title">Browse like no one’s watching.</string>
+    <string name="firstrun_description">Automatically block online trackers while you browse. Then tap to erase visited pages, searches, cookies and passwords from your device.</string>
+    <string name="firstrun_button">OK, GOT IT!</string>
```

Comment:
You do not need to write this in all caps. You can use `android:textAllCaps` on the `Button`. However I think that's already the default on at least the latest Android versions. The advantage of letting Android do it is that it will adapt to whatever makes sense in a language.